docs/dc-apps-performance-toolkit-user-guide-bitbucket.md

```diff
@@ -544,6 +544,8 @@ In the `bitbucket-ui.py` script, view the following block of code:
 # def test_1_selenium_custom_action(webdriver, datasets, screen_shots):
 #     app_specific_action(webdriver, datasets)
 ```
+If you need to run `app_speicifc_action` as specific user uncomment `app_specific_user_login` function in [code example](https://github.com/atlassian/dc-app-performance-toolkit/blob/master/app/extension/bitbucket/extension_ui.py). Note, that in this case `test_1_selenium_custom_action` should follow just before `test_2_selenium_z_log_out` action.
```

Comment:
in bitbucket log out test name is a bit different(oh god why): `test_2_selenium_logout`